### PR TITLE
Release v0.6.0

### PR DESCRIPTION
**Pull request by technoweenie**
Did I miss anything in the changelog? How do the subheaders look? I organized the fixes to break the giant list of changes up.

[rendered](https://github.com/github/git-lfs/blob/b78cd7a4a5107b4a73dc4a29a19b9e46da0eaeba/CHANGELOG.md)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,59 @@
 # Git LFS Changelog
 
+## v0.6.0 (10 September, 2015)
+
+First release with full support for the Batch API, while still supporting
```
Comment (sinbad) [inline CHANGELOG.md]: I think we should say the batch api is now the default?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,59 @@
...
+
+* Docker images and scripts for building and testing linux packages. #511 #526 #555 #603 (@andyneff, @ssgelm)
+* Create Windows GUI installer. #642 (@technoweenie)
+
+### Experimental Features
+
+* Experimental support for Git worktrees (in Git 2.5+) #546 (@sinbad)
+* Experimental extension support. #486 (@ryansimmen)
+
+## v0.5.4 (30 July, 2015)
```
Comment (sinbad) [inline CHANGELOG.md]: Maybe add a mention of the new `git-lfs-config` man page collecting all the settings?

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,59 @@
...
+
+### Fetch
+
+* Rename old `git lfs fetch` command to `git lfs pull`. #527 (@sinbad)
+* Add `git lfs checkout` #527 #543 #551 #566 (@sinbad)
+* Add `git lfs fetch` for _just_ downloading objects. #527 (@sinbad)
+  * Support fetching multiple refs #542 (@sinbad)
+  * Add `--all` option for download all objects from the server. #633 (@sinbad)
+  * Add `--include` and `--exclude` flag for `git lfs fetch` #573 (@sinbad)
+* Fix error handling while `git update-index` is running. #570 (@rubyist)
```
Comment (sinbad) [inline CHANGELOG.md]: Mention `git fetch --recent` here?

Comment (sinbad) [inline CHANGELOG.md]: Also default remote derivation for fetch/pull has slightly changed, it's no longer always `origin`. It follows `git pull` and respects tracking branches.

File: CHANGELOG.md (hunk 2 above)
Comment (sinbad) [inline CHANGELOG.md]: Would be good to mention that binaries are now compiled with Go 1.5 which will fix any authentication issues people might have been having with URLs containing usernames.

**Comment by rubyist** [comment]
v0.6.0 - Sinbad's Revenge!

**Comment by sinbad** [comment]
\*drops smoke pellet\*  \*hides\* :fearful:

**Pull request merged**